person.rb

```diff
@@ -1,32 +1,38 @@
 require_relative 'nameable'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
+require_relative 'rental'
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
Since we are also modifying the rentals here, kindly add getters and setters for the rentals😉.